Feature: spago script (#712)
purescript/spago: README.md

```

### Run a standalone PureScript file as a script

You can run a standalone PureScript file as a script via `spago script`.

> Note: The module name must be `Main`, and it must export an effectful function `main`.

Reviewer: ```suggestion
Note: The module name must be `Main`, and it must export a function `main :: Effect Unit`.
```

By default, the following dependencies are installed: `effect`, `console`, `prelude`.

> You can run the script via the following:
> 
> ```bash
> $ spago script path/to/script.purs
> ```

Reviewer: ```suggestion
You can run a script via the following, optionally specifying a package set to use, and additional dependencies to pull from there:

```bash
$ spago script --tag psc-13.8 -d node-fs path/to/script.purs
```
```
